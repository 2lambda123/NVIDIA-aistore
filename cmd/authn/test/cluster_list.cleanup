ais auth rm cluster srv1
ais auth rm cluster srv0
ais auth rm cluster srv2
