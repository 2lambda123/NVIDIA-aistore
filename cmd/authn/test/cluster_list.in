ais auth login admin admin // IGNORE
ais auth show cluster
ais auth add cluster srv1 http://1.1.1.1
ais auth show cluster
ais auth rm cluster srv0 // FAIL "not found"
ais auth add cluster srv1 http://1.1.1.3 // FAIL "registered"
ais auth show cluster
ais auth add cluster srv0 clu0 http://2.2.2.2
ais auth show cluster
ais auth update cluster srv2 http://3.3.3.3 // FAIL "not found"
ais auth update cluster srv1 http://3.3.3.3
ais auth update cluster srv0 http://2.2.2.2 http://4.4.4.4
ais auth show cluster
