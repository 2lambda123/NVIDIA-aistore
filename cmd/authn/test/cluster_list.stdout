ClusterID       URLs
ClusterID       URLs
srv1            http://1.1.1.1
ClusterID       URLs
srv1            http://1.1.1.1
ClusterID       URLs
srv0            http://2.2.2.2
srv1            http://1.1.1.1
ClusterID       URLs
srv0            http://2.2.2.2,http://4.4.4.4
srv1            http://3.3.3.3
